src/types.ts

```diff
@@ -23,3 +23,49 @@ import { Opaque } from 'type-fest';
 export type AcmeUser = Opaque<any, 'AcmeUser'>;
 export type AcmeGroup = Opaque<any, 'AcmeGroup'>;
 */
```

Comment:
Please remove all of this (unused) boilerplate code + comments associated with it